doc/jsk_perception/nodes/template_matching_detector.rst

```diff
@@ -0,0 +1,93 @@
+TemplateMatchDetector
+=====================
+
+What is this?
+-------------
+
+.. image:: ./images/template_matching_detector.jpg
+
+Detect object by template matching
+
+
+Subscribing Topic
+-----------------
+
+* ``~input`` (``sensor_msgs/Image``)
+
+  Raw image.
+
+* ``~input/info`` (``sensor_msgs/CameraInfo``)
+
+  Camera Info.
+
+
+Publishing Topic
+----------------
+
+* ``~output/rect`` (``jsk_recognition_msgs/RectArray``)
+
+  Rectangles of detected objects
+
+* ``~output/viz`` (``sensor_msgs/Image``)
+
+  Visualized image of detection result.
+
+
+Parameters
+----------
+
+* ``~queue_size`` (Int, default: ``100``)
+
+  Size of queue
```

Comment:
for synchronizer of message_filter.

Reply:
Current template_matching_nodelet is not using message_filter.